pkg/controller/apimanager/apimanager_controller.go

```diff
@@ -138,7 +132,7 @@ func (r *ReconcileAPIManager) upgradeAPIManager(cr *appsv1alpha1.APIManager) (re
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileAPIManager) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	logger := r.Logger().WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	logger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
```

Comment:
Why is log instead of r.Logger() being used?
r *ReconcileAPIManager is a BaseReconciler, and has been instantiated with the `log` variable

Reply:
Right. Upodating it.